### ISSUE TITLE
arange XlaRuntimeError v0.4.3 colab gpu

### ISSUE THREAD

**Issue by abcam** [google/jax#14470]
### Description

jnp.arange causes XlaRuntimeError.

```
!pip install --upgrade "jax[cuda]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```
```
!nvidia-smi
```
```
import jax.numpy as jnp
```
```
x = jnp.arange(33)
```
---------------------------------------------------------------------------

XlaRuntimeError                           Traceback (most recent call last)

[<ipython-input-4-4ed4ff97c1b8>](https://localhost:8080/#) in <module>
----> 1 x = jnp.arange(33)

16 frames

[/usr/local/lib/python3.8/dist-packages/jax/_src/dispatch.py](https://localhost:8080/#) in backend_compile(backend, built_c, options, host_callbacks)
   1024   # TODO(sharadmv): remove this fallback when all backends allow `compile`
   1025   # to take in `host_callbacks`
-> 1026   return backend.compile(built_c, compile_options=options)
   1027 
   1028 _ir_dump_counter = itertools.count()

XlaRuntimeError: INTERNAL: RET_CHECK failure (external/org_tensorflow/tensorflow/compiler/xla/service/gpu/gpu_compiler.cc:626) dnn != nullptr

### What jax/jaxlib version are you using?

v0.4.3

### Which accelerator(s) are you using?

GPU

### Additional system info

Google Colab

### NVIDIA GPU info

Tue Feb 14 16:45:09 2023       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 510.47.03    Driver Version: 510.47.03    CUDA Version: 11.6     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  Tesla T4            Off  | 00000000:00:04.0 Off |                    0 |
| N/A   59C    P0    28W /  70W |      0MiB / 15360MiB |      0%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|  No running processes found                                                 |
+-----------------------------------------------------------------------------+

**Comment by jakevdp** [comment]
Thanks for the question! This looks like an issue of mismatched CUDA versions; see https://github.com/google/jax/issues/14464 for a similar answer.

**Comment by hawkinsp** [comment]
Closing as a duplicate of #14464 ; the fix is to install the correct version of jaxlib.

**Comment by CharlesFauman** [comment]
Hi, I'm seeing the exact same error, on WSL with cuda 11.8. I tried both jaxlib versions.

I also tried updating my cuda to 12.0, but see the same error.

Attempting with fresh install of https://github.com/NTT123/a0-jax if it helps to repro

```
(venv) charlesf@home:~/workspace/a0-jax$ python train_agent.py --weight-decay=1e-2 --num-iterations=3
Cores: [StreamExecutorGpuDevice(id=0, process_index=0, slice_index=0)]
2023-02-14 22:00:43.393817: E external/org_tensorflow/tensorflow/compiler/xla/stream_executor/cuda/cuda_dnn.cc:429] Could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2023-02-14 22:00:43.394357: E external/org_tensorflow/tensorflow/compiler/xla/status_macros.cc:57] INTERNAL: RET_CHECK failure (external/org_tensorflow/tensorflow/compiler/xla/service/gpu/gpu_compiler.cc:626) dnn != nullptr 
*** Begin stack trace ***

















        _PyObject_MakeTpCall

        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault

        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call

        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall


        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault

        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyObject_FastCallDictTstate
        _PyObject_Call_Prepend

        _PyObject_MakeTpCall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        PyObject_Call
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault
        _PyFunction_Vectorcall
        _PyEval_EvalFrameDefault

        PyEval_EvalCode



        _PyRun_SimpleFileObject
        _PyRun_AnyFileObject
        Py_RunMain
        Py_BytesMain
        __libc_start_main
        _start
*** End stack trace ***

Traceback (most recent call last):
  File "/home/charlesf/workspace/a0-jax/train_agent.py", line 330, in <module>
    fire.Fire(train)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/fire/core.py", line 141, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/fire/core.py", line 475, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/fire/core.py", line 691, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "/home/charlesf/workspace/a0-jax/train_agent.py", line 223, in train
    env = import_class(game_class)()
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/pax/_src/core/safe_module.py", line 23, in __call__
    cls.__init__(module, *args, **kwargs)
  File "/home/charlesf/workspace/a0-jax/games/connect_two_game.py", line 52, in __init__
    self.reset()
  File "/home/charlesf/workspace/a0-jax/games/connect_two_game.py", line 67, in reset
    self.board = jnp.zeros((4,), dtype=jnp.int32)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/numpy/lax_numpy.py", line 2126, in zeros
    return lax.full(shape, 0, _jnp_dtype(dtype))
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/lax/lax.py", line 1211, in full
    return broadcast(fill_value, shape)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/lax/lax.py", line 790, in broadcast
    return broadcast_in_dim(operand, tuple(sizes) + np.shape(operand), dims)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/lax/lax.py", line 818, in broadcast_in_dim
    return broadcast_in_dim_p.bind(
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/core.py", line 343, in bind
    return self.bind_with_trace(find_top_trace(args), args, params)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/core.py", line 346, in bind_with_trace
    out = trace.process_primitive(self, map(trace.full_raise, args), params)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/core.py", line 728, in process_primitive
    return primitive.impl(*tracers, **params)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/dispatch.py", line 122, in apply_primitive
    compiled_fun = xla_primitive_callable(prim, *unsafe_map(arg_spec, args),
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/util.py", line 253, in wrapper
    return cached(config._trace_context(), *args, **kwargs)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/util.py", line 246, in cached
    return f(*args, **kwargs)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/dispatch.py", line 201, in xla_primitive_callable
    compiled = _xla_callable_uncached(lu.wrap_init(prim_fun), device, None,
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/dispatch.py", line 354, in _xla_callable_uncached
    return computation.compile(_allow_propagation_to_outputs=allow_prop).unsafe_call
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 3203, in compile
    executable = self._compile_unloaded(
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 3174, in _compile_unloaded
    return UnloadedMeshExecutable.from_hlo(
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/interpreters/pxla.py", line 3456, in from_hlo
    xla_executable = dispatch.compile_or_get_cached(
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/dispatch.py", line 1081, in compile_or_get_cached
    return backend_compile(backend, serialized_computation, compile_options,
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/profiler.py", line 314, in wrapper
    return func(*args, **kwargs)
  File "/home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages/jax/_src/dispatch.py", line 1026, in backend_compile
    return backend.compile(built_c, compile_options=options)
jaxlib.xla_extension.XlaRuntimeError: INTERNAL: RET_CHECK failure (external/org_tensorflow/tensorflow/compiler/xla/service/gpu/gpu_compiler.cc:626) dnn != nullptr 
(venv) charlesf@home:~/workspace/a0-jax$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2022 NVIDIA Corporation
Built on Mon_Oct_24_19:12:58_PDT_2022
Cuda compilation tools, release 12.0, V12.0.76
Build cuda_12.0.r12.0/compiler.31968024_0
(venv) charlesf@home:~/workspace/a0-jax$ pip show jaxlib
Name: jaxlib
Version: 0.4.3+cuda11.cudnn86
Summary: XLA library for JAX
Home-page: https://github.com/google/jax
Author: JAX team
Author-email: jax-dev@google.com
License: Apache-2.0
Location: /home/charlesf/workspace/a0-jax/venv/lib/python3.10/site-packages
Requires: numpy, scipy
Required-by: chex, mctx, optax
(venv) charlesf@home:~/workspace/a0-jax$ 
```

**Comment by CharlesFauman** [comment]
I don't get this error if i install 0.4.2 of jax[cuda] with:
```
pip install --upgrade "jax[cuda]==0.4.2" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```

**Comment by abcam** [comment]
> Thanks for the question! This looks like an issue of mismatched CUDA versions; see #14464 for a similar answer.

Thank you!

**Comment by hawkinsp** [comment]
@CharlesFauman I suspect you don't have the correct version of CuDNN installed.
